CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+* fix!: EXPOSED-536 Short column allows out-of-range values by @joc-a in https://github.com/JetBrains/Exposed/pull/2231
+* fix!: EXPOSED-482 Cannot use `Column.transform()` to return null by @obabichevjb in https://github.com/JetBrains/Exposed/pull/2235
+* fix!: EXPOSED-546 [H2] Column with Json default value can not be used as databaseGenerated by @obabichevjb in https://github.com/JetBrains/Exposed/pull/2241
+
+Deprecations:
+* deprecate!: EXPOSED-550 DeleteStatement holds unused offset property by @bog-walk in https://github.com/JetBrains/Exposed/pull/2243
+
+Features:
+* feat: EXPOSED-498 Handle auto-increment status change on a column by @joc-a in https://github.com/JetBrains/Exposed/pull/2216
+* feat: EXPOSED-552 Include DROP statements for unmapped columns for migration by @joc-a in https://github.com/JetBrains/Exposed/pull/2249
+
+Bug fixes:
+* fix: EXPOSED-513 DROP SEQUENCE fails when there is a dot in the sequence name by @joc-a in https://github.com/JetBrains/Exposed/pull/2220
+* fix: EXPOSED-496 reference() idColumn equality check with referree's id is insufficient by @bog-walk in https://github.com/JetBrains/Exposed/pull/2222
+* fix: EXPOSED-528 Escape parameter placeholder '?' by double question mark '??' by @JajaComp in https://github.com/JetBrains/Exposed/pull/2227
+* fix: EXPOSED-509 Upsert with escaped multiline string fails in prepared statement by @bog-walk in https://github.com/JetBrains/Exposed/pull/2224
+* fix: EXPOSED-495 Unable to create new Entity when server-side default valu… by @obabichevjb in https://github.com/JetBrains/Exposed/pull/2240
+* fix: EXPOSED-547 idParam() registers composite id value with a single placeholder by @bog-walk in https://github.com/JetBrains/Exposed/pull/2242
+* fix: EXPOSED-527 BUG: mergeFrom(...) using a query with const-condition do… by @obabichevjb in https://github.com/JetBrains/Exposed/pull/2236
+* fix: EXPOSED-551 [SQL Server] Update from join with limit throws syntax exception by @bog-walk in https://github.com/JetBrains/Exposed/pull/2244
+* fix: EXPOSED-558 Entity cache for upsertReturning statements results in stale return values by @rasharab in https://github.com/JetBrains/Exposed/pull/2248
+* fix: EXPOSED-373 Close ResultSet before closing Statement to suppress Agroal leak warning by @ivan-gomes in https://github.com/JetBrains/Exposed/pull/2247
+* fix: EXPOSED-562 Any caught exception from inner transaction triggers full rollback by @bog-walk in https://github.com/JetBrains/Exposed/pull/2251
+
+Docs:
+* docs: Update documentation website by @github-actions in https://github.com/JetBrains/Exposed/pull/2221
+* docs: EXPOSED-515 How to identify composite key columns that use reference() by @bog-walk in https://github.com/JetBrains/Exposed/pull/2225
+
+Chore:
+* chore: Remove docker yml files warning -> the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion by @joc-a in https://github.com/JetBrains/Exposed/pull/2232
```

Comment:
We don't usually include any commits marked by `chore` (unless its an outside contribution). Especially since this change only affects our test setup and isn't felt by user.